### PR TITLE
add documentation for sympy.sets.fancysets.Reals

### PR DESCRIPTION
**Pull request by highpost**
#### Brief description of what is fixed or changed
The sympy.sets.fancysets.Reals class has no documentation, so this is a small documentation-only PR to fill that void.

#### Release Notes

* sets
  * Added documentation and examples for sympy.sets.fancysets.Reals.

### CODE REVIEW

**Review by sylee957 — APPROVED** [review]
Looks good.

But I think that `pprint` can be unnecessary here

File: sympy/sets/fancysets.py [hunk 1]
```diff
@@ -148,7 +148,34 @@ def _boundary(self):
 
 
 class Reals(with_metaclass(Singleton, Interval)):
+    """
+    Represents all continuous numbers
```
Comment (smichr) [inline sympy/sets/fancysets.py]: Would it be good to add "finite"?

File: sympy/sets/fancysets.py [hunk 2]
```diff
@@ -148,7 +148,34 @@ def _boundary(self):
...
+    from negative infinity to positive infinity,
+    including all integer, rational and irrational numbers.
+    This set is also available as the Singleton, S.Reals.
+
+
+    Examples
+    ========
+
+    >>> from sympy import S, Interval, Rational, pi, I, pprint
+    >>> pprint(5 in S.Reals)
```
Comment (smichr) [inline sympy/sets/fancysets.py]: these look the same without pprint for me -- do they look different to you?

File: sympy/sets/fancysets.py (hunk 1 above)
Comment (jksuom) [reply sympy/sets/fancysets.py]: The meaning of "continuous numbers" is obscure. You probably mean "real numbers".

**Comment by oscarbenjamin** [comment]
Thanks for this!

I'll merge now

**Pull request merged**